android/AndroidManifest.xml

```diff
@@ -56,7 +58,6 @@
             <intent-filter>
                 <action android:name="android.hardware.usb.action.USB_DEVICE_ATTACHED" />
                 <action android:name="android.hardware.usb.action.USB_DEVICE_DETACHED" />
-                <action android:name="android.hardware.usb.action.USB_ACCESSORY_ATTACHED" />
```

Comment:
What's the distinction between USB_DEVICE and USB_ACCESORY?

Reply:
Accessory was used for Taisync before that was removed. It's just which end acts as the USB Host. https://developer.android.com/develop/connectivity/usb